### PR TITLE
Fix: Get Ads by DateRange

### PR DESCRIPTION
**Pull request by Maksym82564**
1. Spring Data JPA was unable to parse the original method name in the repository class, so the method name was changed.
2. API endpoint was returning null for the advert creator ID, because ModelMapper couldn't automatically convert User entity to Long (creator ID). Custom converter was added to ModelMapper to resolve this issue.

### CODE REVIEW

File: src/main/java/com/codegym/jrugotom5/config/AppConfig.java [hunk 1]
```diff
@@ -8,6 +9,13 @@
 public class AppConfig {
     @Bean
     public ModelMapper modelMapper() {
-        return new ModelMapper();
+        ModelMapper modelMapper = new ModelMapper();
+        modelMapper.addConverter(advertToDtoConverter());
```
Comment (Volodymyr95) [inline src/main/java/com/codegym/jrugotom5/config/AppConfig.java]: why u added it here?

File: src/test/java/com/codegym/jrugotom5/service/AdvertServiceTest.java [hunk 2]
```diff
@@ -57,7 +57,7 @@ void testGetAdvertsByDateRange() {
         advertDTO2.setId(2L);
         advertDTO2.setTitle("Advert 2");
 
-        when(advertRepository.findAllByCreatedDateBetweenFromTo(from, to)).thenReturn(adverts);
+        when(advertRepository.findAllByCreatedDateBetween(from, to)).thenReturn(adverts);
         when(modelMapper.map(advert1, AdvertDTO.class)).thenReturn(advertDTO1);
```
Comment (Volodymyr95) [inline src/test/java/com/codegym/jrugotom5/service/AdvertServiceTest.java]: if u will have 100 item, u will add 100 when block?

File: src/main/java/com/codegym/jrugotom5/config/AppConfig.java (hunk 1 above)
Comment (Maksym82564) [reply src/main/java/com/codegym/jrugotom5/config/AppConfig.java:13]: it is a creation of model mapper bean, where else i should add it

File: src/test/java/com/codegym/jrugotom5/service/AdvertServiceTest.java (hunk 2 above)
Comment (Maksym82564) [reply src/test/java/com/codegym/jrugotom5/service/AdvertServiceTest.java]: rewrote this test

File: src/main/java/com/codegym/jrugotom5/config/AppConfig.java (hunk 1 above)
Comment (Maksym82564) [reply src/main/java/com/codegym/jrugotom5/config/AppConfig.java]: ended up completly removing this converter and performing conversion without it

**Review by Volodymyr95 — APPROVED** [review]

**Pull request merged**